apps/src/templates/teacherDashboard/TeacherDashboardNavigation.story.jsx

```diff
@@ -2,38 +2,39 @@ import React from 'react';
 import {BrowserRouter as Router, Route} from 'react-router-dom';
 import TeacherDashboardNavigation from './TeacherDashboardNavigation';
 
-export default storybook => {
-  storybook.storiesOf('TeacherDashboardNavigation', module).addStoryTable([
-    {
-      name: 'Scrollable navigation',
-      description:
-        'Navbar should scroll when tabs are too large to fit horizontally',
-      story: () => {
-        return (
-          <Router basename="/">
-            <Route component={props => <TeacherDashboardNavigation />} />
-          </Router>
-        );
-      }
-    },
-    {
-      name: 'Non-scrollable navigation',
-      description: 'Navbar should not scroll when tabs fit horizontally',
-      story: () => {
-        const links = [
-          {label: 'Progress', url: '/progress'},
-          {label: 'Stats', url: '/stats'},
-          {label: 'Students', url: '/students'}
-        ];
+export default {
+  name: 'TeacherDashboardNavigation',
+  component: TeacherDashboardNavigation
+};
+
+export const ScrollableNavigation = () => {
+  const links = [];
+
+  for (let i = 0; i < 20; i++) {
```

Comment:
oh oops, meant to add a comment. the reason I added this is because the old `Scrollable navigation` test didn't actually have enough tabs to scroll through. So I dynamically added 20 to ensure that we could scroll (like you intuited). I'll add a comment here but also open to suggestions on how to do this better!

Reply:
Comment is perfect - I just wanted to make sure I wasn't missing something with this transition!